src/configuration/app-containers.md

```diff
@@ -211,37 +212,37 @@ web:
 #### Upstream
 
 `upstream` is an optional key that describes how your application listens to
-requests, and what protocol it speaks.
+requests and what protocol it speaks.
 
 The following subkeys can be defined:
 * `socket_family`:
     Default: `tcp`. Describes whether your application will listen on a
-    Unix socket or a TCP socket. Can be either `tcp` or `unix`.
+    Unix socket or a TCP socket and can be either `tcp` or `unix`.
 * `protocol`:
     Specifies whether your application is going to receive incoming requests
-    over HTTP, FastCGI, or UWSGI. Can be `http` or `fastcgi`, for now. The
+    over HTTP, FastCGI, or UWSGI and can be `http` or `fastcgi`, for now. The
```

Comment:
How about this?  ... over HTTP (`http`), FastCGI (`fastcgi`) or UWSGI (not available currently).